### DISCUSSION TITLE
Any recommendations to style the embed/ link menu?

### DISCUSSION THREAD

**Discussion by rmbe41** [jpuri/react-draft-wysiwyg#1137] [Q&A]
First of all, thanks so much for this awesome editor - it couldn't be a better fit for our project :)
The challenge I am currently facing is how to style menus located in the toolbar like inserting a link or embedding a video see screenshot? Are there any best practices or recommendations?

![image](https://user-images.githubusercontent.com/63474956/121654999-90ec9780-ca9e-11eb-932b-23ca4e3c87bb.png)

**Comment by boneyrox** [comment]
Hi,
Can you try by targeting class: `.rdw-embedded-modal` and add `overflow-y:auto` property to it
